sqlmesh/core/console.py

```diff
@@ -131,15 +131,31 @@ def stop_promotion_progress(self, success: bool = True) -> None:
 
     @abc.abstractmethod
     def start_migration_progress(self, total_tasks: int) -> None:
-        """Indicates that a new migration progress has begun."""
+        """Indicates that a new snapshot migration progress has begun."""
 
     @abc.abstractmethod
     def update_migration_progress(self, num_tasks: int) -> None:
```

Comment:
Let's update these method names too, like we did for `stop_migration_progress` -> `stop_snapshot_migration_process`.